src/anupSJun20/StaticNonstatic.java

```diff
@@ -0,0 +1,63 @@
+package anupSJun20;
+/*Design to test static, non static, return type concept.
+
+1) Method sum(), accept 3 parameters of type double & return answer
+2) Method average(), accept 3 parameters of type double & return answer
+3) Method isAnswerAboveExpecation(), take one parameter (answer which you received from average method) and return true if value is more then 50 else false.
+4) Method isEligible(),take one parameter (answer which you received from average method) return true if value is more then 50 or divisible by 2. Eligibility will be printed in main method. 
+5) Method getYourGrade(),take one parameter (answer which you received from average method) return "A Grade" if average is more then 80 else "B grade", main method will print grade received from getYourGrade() method.
+
+Rules : a) From main method, you will call average() and average() method will call sum() method.
+               Main method will not directly call sum method. 
+	    b) average() & sum() should be non static method.
+	    c) isAnswerAboveExpecation(), isEligible(), getYourGrade() method should be static method and      called from main method.
+            d) Class should not have any instance variable. */
+
+public class StaticNonstatic {
+	double sum(double subject1,double subject2, double subject3) {
+		return (subject1+subject2+subject3);
+	}
+	double average(double subject1,double subject2, double subject3) {
+		sum(subject1,subject2, subject3);
```

Comment:
Here calling of sum() method does not make sense. Because in the next line you are using it.

Reply:
Changes incorporated.